x-pack/plugins/ml/public/application/routing/routes/data_frame_analytics/analytics_job_exploration.tsx

```diff
@@ -38,16 +39,31 @@ const PageWrapper: FC<PageProps> = ({ location, deps }) => {
   const { context } = useResolver('', undefined, deps.config, basicResolvers(deps));
   const { _g }: Record<string, any> = parse(location.search, { sort: false });
 
+  const urlGenerator = useMlUrlGenerator();
+  const {
+    services: {
+      application: { navigateToUrl },
+    },
+  } = useMlKibana();
+
+  const redirectToAnalyticsManagementPage = async () => {
+    const url = await urlGenerator.createUrl({ page: ML_PAGES.DATA_FRAME_ANALYTICS_JOBS_MANAGE });
+    await navigateToUrl(url);
+  };
+
   let globalState: any = null;
   try {
     globalState = decode(_g);
   } catch (error) {
     // eslint-disable-next-line no-console
-    console.error('Could not parse global state');
-    window.location.href = '#data_frame_analytics';
+    console.error(
+      'Could not parse global state. Redirecting to Dataframe Analytics Management Page.'
```

Comment:
Nit as this is only shown in the console, but we use two words for `Data Frame`.

Reply:
Updated here https://github.com/elastic/kibana/pull/76735/commits/dc3ba76792563da7c40db99ac11c6d69100f3bd7